recipes/blake2/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+
+class blake2Conan(ConanFile):
+    name = "blake2"
+    version = "997fa5b" #BLAKE2 doesn't have versions so we use the commit hash instead
+    license = "	CC0-1.0, OpenSSL, APSL 2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/BLAKE2/BLAKE2"
+    description = ("BLAKE2 is a cryptographic hash function faster than MD5, \
+                    SHA-1, SHA-2, and SHA-3, yet is at least as secure as the latest standard SHA-3")
+    settings = "os", "arch", "compiler", "build_type"
+    topics = ("conan", "blake2", "hash")
+    exports_sources = ["makefile"]
+    options = {"fPIC": [True, False], "use_sse": [True, False]}
+    default_options = {"fPIC": True, "use_sse": True}
+    _source_subfolder = "source_subfolder"
+    _env_build = None
+    _env_vars = None
+
+
+    def _configure_envbuilds(self):
```

Comment:
Does it work on Windows and Apple?

Reply:
If you're asking about the library, I don't really know. We only use it on Linux and the documentation is extremely poor.

Reply:
I would say to ask the author first, because trying to build on Windows and MacOS just to check it is a pain.

Reply:
It already passed all the builds on the CI so it seems to work.